### PR TITLE
Fix: The search in the multi select is not showing + Multi select dropdown stops functioning if default value is undefined

### LINKED ISSUE #2051: [Bug] The search in the multi select is not showing
### Steps to reproduce the behaviour:
1. Drag a dropdown
2. Convert to multi select
3. Enter text in the multi select to filter options
4. The options get filtered but the text is not visible

### PR DESCRIPTION
**Pull request by jsartisan**
## Description
- For the multi-select widget, the ghost input width was just 1px and was not visible at all. Increased the width for it to make it visible.
- The dropdown does not work properly when default option is undefined for multi select.

Fixes #2051 & #2052 

## Type of change
- Bugfix

## How Has This Been Tested?
- Manually tested
- Cypress tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/components/designSystems/blueprint/DropdownComponent.tsx [hunk 1]
```diff
@@ -216,7 +216,7 @@ const StyledMultiDropDown = styled(MultiDropDown)<{
       .${Classes.INPUT_GHOST} {
         flex: 0 0 auto;
         margin: 0;
-        width: 1px;
+        width: 60px;
```
Comment (hetunandu) [inline app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:219]: This values should be derived from the width of the widget

**Review by hetunandu — CHANGES_REQUESTED** [review]
Minor changes

File: app/client/src/components/designSystems/blueprint/DropdownComponent.tsx (hunk 1 above)
Comment (jsartisan) [reply app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:219]: This is not the widget of the widget. This is actually the widget of the ghost text input inside the select box.

Comment (hetunandu) [reply app/client/src/components/designSystems/blueprint/DropdownComponent.tsx:219]: Limiting this width is causing it to only show a bit of the input text and the rest is going out of the screen. For this reason it should take account of the available width and then set this width.

https://www.loom.com/share/3e648dfe88fd496c8ceb196d9014d160

**Comment by jsartisan** [comment]
@hetunandu Updated styling according to feedback.

**Review by hetunandu — APPROVED** [review]

**Pull request merged**